tests/queries/0_stateless/02676_trailing_commas.sql

```diff
@@ -0,0 +1,5 @@
+SELECT 1,;
+SELECT 1, FROM numbers(1);
+WITH 1 as a SELECT a, FROM numbers(1);
+WITH 1 as from SELECT from, from + from, from in [0], FROM numbers(1);
+SELECT n, FROM (SELECT 1 AS n);
```

Comment:
maybe makes sense to add also some scalar subquery (like `(select * from numbers(1))`)
also smth like `select [1, 2, from numbers(10)` just in case.